component/vshn_appcat_services.jsonnet

```diff
@@ -187,6 +223,7 @@ local vshn_appcat_service(name, serviceParams) =
     ['20_plans_vshn_%s' % name]: plansCM,
     ['22_prom_rule_sla_%s' % name]: promRuleSLA,
     [if isOpenshift && std.objectHas(serviceParams, 'openshiftTemplate') then '21_openshift_template_%s_vshn' % name]: osTemplate,
+    [if isOpenshift then '22_scc_appcat']: scc,
```

Comment:
Can you move this out of the generic `vshn-services` logic? Right now it will always deploy this specific collabora manifest if it's openshift, even if collabora is not enabled on a cluster. I'd probably put it in a `vshn_nextcloud.jsonnet` file.

Reply:
I just added additional variable to if statement to verify whether nextcloud is enabled or not. separate file to just handle scc seems like an overengineering

Reply:
I just switched logic to separate vshnnextcloud.jsonnet file.

Reply:
I agree with @Kidswiss, the `scc` file should go to `vshn-nextcloud.jsonnet`. It's unfortunate that we don't have it for nextcloud yet but we have a `.jsonnet` for every other service. It's time to give nextcloud it's own file.

Reply:
It's already in separate file.